packages/sku/lib/storybook.js

```diff
@@ -0,0 +1,70 @@
+const path = require('path');
+
+const { paths } = require('../context');
+const fs = require('fs/promises');
+const debug = require('debug');
+const glob = require('fast-glob');
+
+const log = debug('sku:storybook');
+
+const previewFileName = 'preview.{js,ts,tsx}';
+
+const previewFileAbsolutePath = path.join(
+  path.dirname(paths.appSkuConfigPath),
+  '.storybook',
+  previewFileName,
+);
+
+/**
+ * This function first cleans up any existing preview.{js,ts,tsx} files in the provided storybook config directory.
+ * This includes any potentially dangling symlinks.
+ * Then it looks for a `.storybook/preview.{js,ts,tsx}` file relative to the sku config folder.
+ * If 1 file is found, a symlink is created to it from the provided storybook config directory.
+ * Does nothing if 0 or >1 files are found.
+ *
+ * @param {string} storybookConfigDirectory The path to the storybook config directory
+ * */
+const setUpStorybookPreviewFile = async (storybookConfigDirectory) => {
+  const symlinkFileAsbolutePath = path.join(
+    storybookConfigDirectory,
+    previewFileName,
+  );
+  const symlinkFiles = await glob(symlinkFileAsbolutePath, {
+    dot: true,
+    onlyFiles: false, // Required to find dangling symlinks
+  });
+
+  // Clean up any existing files in the storybook config folder so we have a clean slate
+  for (const symlinkFile of symlinkFiles) {
```

Comment:
So if >1 files are found, this would remove all symlinks but not create any symlinks in place. Is that intentional?

Reply:
Yes. Since it's globbing for multiple file extensions, it could potentially return multiple files. However, if you change from `preview.js` to `preview.tsx` for example, the next time you run `sku storybook`, it would remove the old symlink and then make the new one, so I don't even think it's possible to end up with multiple symlinks unless you manually add one yourself.

Reply:
It can happen you have `preview.js` and `preview.tsx` because of a bad merge. Very edgy case.